src/views/catalog/templatescatalog/hooks/useIsTemplateSupportsQuickCreate.ts

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+
+import { V1Template } from '@kubevirt-ui/kubevirt-api/console';
+import { k8sCreate } from '@openshift-console/dynamic-plugin-sdk';
+import { K8sModel } from '@openshift-console/dynamic-plugin-sdk/lib/api/common-types';
+
+export const ProcessedTemplatesModel: K8sModel = {
+  label: 'Processed Template',
+  labelPlural: 'Processed Templates',
+  apiVersion: 'v1',
+  apiGroup: 'template.openshift.io',
+  plural: 'processedtemplates',
+  abbr: 'PT',
+  namespaced: true,
+  kind: 'Template',
+};
+
+export const useIsTemplateSupportsQuickCreate = (template: V1Template): [boolean, boolean, any] => {
```

Comment:
It will be nice to have this hook return the VM to be crated instead of just if an error happened
the name can then be more informative, it will tell what we actullay doing ( useProcessedTemplate ? )
and the user will use the return error as indicator if the template is ready to be created as is, WDYT ?